src/test/routes/receiver.ts

```diff
@@ -97,6 +100,22 @@ describe('Login receiver', async () => {
         })
       })
 
+      context('when defendant has one or more draft responses', async () => {
+
+        it('should redirect to response task-list', async () => {
```

Comment:
Test case lies here, it says redirect to task list while in fact we expect redirect to dashboard.